### ISSUE TITLE
Allow downloading via https

### ISSUE THREAD

**Issue by perrin4869** [99x/dynamodb-localhost#62]
Behind a firewall, `http` could be disabled in some usecases... it would be nice to be able to support downloading via https as well

**Comment by mrloop** [comment]
And now it looks like `http` is no longer supported, you must download from `https`

**Comment by mrloop** [comment]
https://github.com/mrloop/dynamodb-localhost/commit/e1de7ebc0b505a0a8e67ccf1cfc2ae66021bfc0b

**Comment by msesun** [comment]
Seeing same issue as @mrloop

**Comment by msesun** [comment]
@AshanFernando @henhal  I think there's already a PR that resolves this issue https://github.com/99x/dynamodb-localhost/pull/70

**Comment by perrin4869** [comment]
Yeah my CI finally failed yesterday... hopefully we can get a patch from @AshanFernando  over the weekend or we'll have to `patch-package` this...

**Comment by mrloop** [comment]
Package seems under maintained don't know how likely that is :(, will need to continue using a fork.

**Comment by perrin4869** [comment]
Yeah, but since this will basically break anyone using this package it might get enough people to chime in that it actually gets fixed haha
It's undermaintained but it did have commits 2 months ago
I hope they release version 1.0.0 so they can take advantage of semver

**Comment by domdomegg** [comment]
FYI: This package (`dynamodb-localhost`) doesn't appear to be maintained, with no new releases to NPM in over 4 years.

**Solution:** You can use [aws-dynamodb-local](https://www.npmjs.com/package/aws-dynamodb-local), a maintained fork, instead. (Disclaimer: I am a contributor to this fork). It is a drop-in replacement for this package, and is updated to fix this bug.

Migrating takes about 2 minutes, with [a full guide in the README](https://github.com/raisenational/aws-dynamodb-local#migrating-from-dynamodb-localhost). Of course, it's all still [open-source](https://github.com/raisenational/aws-dynamodb-local) and MIT licensed. Ownership of this new package sits with a registered charity, that is committed to maintaining the package into the future and is open to contributions from the community.
